airtable/request.js

```diff
@@ -86,6 +86,15 @@ export const getAllOwners = async (filterByFormula = '', sort = []) => {
   return getAllRecords(Tables.Owner, filterByFormula, sort);
 };
 
+export const getOwnersByEmail = async (value, sort = []) => {
```

Comment:
might be a good idea to post-fix this with unsafe e.g. getOwnersByEmailUnsafe or otherwise indicate that this is a potentially dangerous function... 

Also some comments about why  we need to do this (limitations of airlock) might be helpful for the future too

Reply:
why is this potentially unsafe? This is on the backend so it doesn’t go through airlock

Reply:
unsafe as in to denote that this function can potentially return the data of every single person in the database, but also I'm not hung up on this